mslib/msui/wms_control.py

```diff
@@ -474,6 +474,7 @@ def __init__(self, parent=None, default_WMS=None, wms_cache=None, view=None):
             self.wms_cache = None
 
         # Initialise date/time fields with current day, 00 UTC.
+        # To do Before refactoring to an aware datetime object add a test to verify the WMS part.
         self.dteInitTime.setDateTime(QtCore.QDateTime(
             datetime.utcnow().replace(hour=0, minute=0, second=0, microsecond=0)))
```

Comment:
this might also become datetime.now() with UTC 


nice article by Miguel
https://blog.miguelgrinberg.com/post/it-s-time-for-a-change-datetime-utcnow-is-now-deprecated

Reply:
I've overseen that this again is wms_control. We don't touch it.

Reply:
AwareDateTime is fine